powerscale/provider/cluster_time_resource.go

```diff
@@ -0,0 +1,258 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"terraform-provider-powerscale/client"
+	"terraform-provider-powerscale/powerscale/helper"
+	"terraform-provider-powerscale/powerscale/models"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+// Ensure provider defined types fully satisfy framework interfaces.
+var (
+	_ resource.Resource                = &ClusterTimeResource{}
+	_ resource.ResourceWithConfigure   = &ClusterTimeResource{}
+	_ resource.ResourceWithImportState = &ClusterTimeResource{}
+)
+
+// NewClusterTimeResource creates a new resource.
+func NewClusterTimeResource() resource.Resource {
+	return &ClusterTimeResource{}
+}
+
+// ClusterTimeResource defines the resource implementation.
+type ClusterTimeResource struct {
+	client *client.Client
+}
+
+// Metadata describes the resource arguments.
+func (r *ClusterTimeResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_cluster_time"
+}
+
+// Schema describes the resource arguments.
+func (r *ClusterTimeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "This resource is used to manage the Cluster Time Settings entity of PowerScale Array. " +
+			"PowerScale Cluster Time Settings provide the ability to configure owner settings on the cluster." +
+			"We can Create, Update and Delete the Cluster Time Settings using this resource. We can also import existing Cluster Time Settings from PowerScale array. " +
+			"Note that, Cluster Time Settings is the native functionality of PowerScale. When creating the resource, we actually load Cluster Time Settings from PowerScale to the resource state. ",
+		Description: "This resource is used to manage the Cluster Time Settings entity of PowerScale Array. " +
+			"PowerScale Cluster Time Settings provide the ability to configure Time settings on the cluster." +
+			"We can Create, Update and Delete the Cluster Time Settings using this resource. We can also import existing Cluster Time Settings from PowerScale array. " +
+			"Note that, Cluster Time Settings is the native functionality of PowerScale. When creating the resource, we actually load Cluster Time Settings from PowerScale to the resource state. ",
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
```

Comment:
Please check if the description is matching with the value getting stored

Reply:
Updated